Minh/taskNavigationComponent_AndroidJetPack/app/src/main/java/com/example/demo/ui/adapter/ItemAdapter.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.demo.ui.adapter
+
+import android.annotation.SuppressLint
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.demo.data.model.SuggestedSearches
+import com.example.demo.databinding.ItemDataBinding
+
+class ItemAdapter : RecyclerView.Adapter<ItemAdapter.ViewHolder>() {
+    private var list: MutableList<SuggestedSearches> = mutableListOf()
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ItemAdapter.ViewHolder {
+        val binding=ItemDataBinding.inflate(LayoutInflater.from(parent.context),parent, false)
+        return ViewHolder(binding)
+    }
+
+    @SuppressLint("NotifyDataSetChanged")
+    fun setData(suggestedSearches: List<SuggestedSearches>){
+        list.clear()
```

Comment:
Tìm hiểu và ap dụng DIFFCallback 
val list = AsyncListDiffer(this, DiffCallback<SuggestedSearches>())

 fun updateItems(items: List<SuggestedSearches>) {
        this.items.submitList(items)
    }